src/app/containers/MediaPlayer/helpers/metadata/index.js

```diff
@@ -4,7 +4,15 @@ import pathOr from 'ramda/src/pathOr';
 const DEFAULT_IMAGE_RES = '1024x576';
 
 export const getThumbnailUri = aresMetadataBlock => {
-  const imageUrl = pathOr('', ['model', 'imageUrl'], aresMetadataBlock);
+  let imageUrl = pathOr('', ['model', 'imageUrl'], aresMetadataBlock);
```

Comment:
Is it worth trying out the the 'url-parse' library to make this a little bit simpler? See https://github.com/bbc/simorgh/blob/76b89386a36e51298d889be6f208d37fa4544469/src/app/lib/utilities/urlParser/index.js#L5

```
const imageUrl = pathOr('', ['model', 'imageUrl'], aresMetadataBlock);

if (imageUrl) {
  const thumbnailUrl = imageUrl.replace('$recipe', DEFAULT_IMAGE_RES);
  let url = new Url(thumbnailUrl);
  url.protocol = 'https';
  return url.toString();
}
```

(Although I have technically broken rules of immutability with the `let url =`.. statement

Reply:
So this approach conflicts with a few (13) snapshots for local fixtures: the url-parse library seems to prefix the url with the host/host name, so when using it you start to see urls prefixed with 'localhost'. i.e. 

expected : `https://ichef.test.bbci.co.uk/images/ic/1024x576/p01k6mtv.jpg`
received : `https://localhost/ichef.test.bbci.co.uk/images/ic/1024x576/p01k6mtv.jpg`

Looking at methods of eliminating this you either end up with no '//' after https. i.e.
```
const url = new Url(imageUrl, {});
// https:ichef.test.bbci.co.uk/images/ic/1024x576/p01k6mtv.jpg
```

or an extra '/' i.e.
```
const url = new Url(imageUrl);
url.host = '';
// https:///ichef.test.bbci.co.uk/images/ic/1024x576/p01mr7dp.jpg;
```

Not sure if you know a way around this? If not I'll just stick with the PR's implementation.

Reply:
Na, leave it. It's fine as is.